static/js/stream_list.js

```diff
@@ -402,12 +512,8 @@ exports.rename_stream = function (sub) {
 
 $(function () {
     $(document).on('narrow_activated.zulip', function (event) {
-        if (zoomed_to_topics && (active_stream_name() !== zoomed_stream)) {
-            zoom_out();
-        }
 
-        $("ul.filters li").removeClass('active-filter active-subject-filter');
-        remove_expanded_subjects();
+        reset_to_unnarrowed();
```

Comment:
Because if you're zoomed to topics, the existing stream list is hidden by the zoom popover and there's no way to even select a different stream.